lib/pool-management/lambdas/forward-test-results.ts

```diff
@@ -0,0 +1,71 @@
+import { captureAWS } from "aws-xray-sdk";
+import * as AWS from "aws-sdk";
+
+interface ForwardTestResultEvent {
+  readonly tenant: string;
+  readonly poolName: string;
+  readonly testResult: string;
+}
+
+captureAWS(AWS);
+
+export const handler = async (event: ForwardTestResultEvent) => {
+  const env = validateEnvironmentVars();
+  const testPool = await readTestPoolFromDynamoDb(env.poolTableName, event.poolName, event.tenant);
+
+  const testResult = event.testResult === "N" ? "NEGATIV" : "POSITIV";
+  const messageBody = `Gruppe ${event.poolName} - SARS-CoV-2 Pool-Testergebnis: ${testResult}`;
+  const date = new Date();
+  const campaignName = `POOL_TEST_FWD_${event.poolName}_${date.getFullYear()}-${date.getMonth()}-${date.getDate()}`;
```

Comment:
`new Date().toISOString().split('T')[0]` might get us a more uniformly formatted date.
The current approach would today create `2021-4-4`.

Reply:
That's cool. I created a new issue for this. 
Campaign names are not unique so it is valid to have multiple campaigns with the same name. We could also argue to remove the date logic completely and set the name to something like "POOL_TEST_FWD_tenant_poolName"